CesiumGltf/include/CesiumGltf/PropertyTablePropertyViewTypes.h

```diff
@@ -0,0 +1,294 @@
+#pragma once
+
+#include "CesiumGltf/PropertyTablePropertyView.h"
+
+#include <glm/common.hpp>
+
+#include <algorithm>
+#include <cstdint>
+#include <string>
+#include <string_view>
+#include <variant>
+
+namespace CesiumGltf {
+
+#define View PropertyTablePropertyView
+
+/**
+ * @brief An alias for all of the potential types of
+ * {@link PropertyTablePropertyView} possible without normalization.
+ * Can be useful for applications that want to implement abstract
+ * representations of PropertyTablePropertyView, without the mess of templated
+ * types.
+ */
+typedef std::variant<
```

Comment:
I added this anticipating that we'll need this `typedef` for Unreal / Unity, but it's not used anywhere in Native right now